array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,14 @@
+/* exported initial */
+/* receive argument array
+iterate through each element of the array and add each element except the last to a new array
+return new array */
+
+function initial(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== (array.length - 1)) {
```

Comment:
Instead of having a conditional run with every iteration, you should just stop the loop one step earlier.